astropy/config/tests/test_configs.py

```diff
@@ -14,74 +15,108 @@
 from astropy.config import configuration
 from astropy.config import paths
 from astropy.utils.exceptions import AstropyDeprecationWarning
+from astropy.config.configuration import get_config, reload_config
 
 
 def test_paths():
     assert 'astropy' in paths.get_config_dir()
     assert 'astropy' in paths.get_cache_dir()
 
+    assert 'testpkg' in paths.get_config_dir(rootname='testpkg')
+    assert 'testpkg' in paths.get_cache_dir(rootname='testpkg')
+
 
 def test_set_temp_config(tmpdir, monkeypatch):
     monkeypatch.setattr(paths.set_temp_config, '_temp_path', None)
 
-    orig_config_dir = paths.get_config_dir()
+    orig_config_dir = paths.get_config_dir(rootname='astropy')
     temp_config_dir = str(tmpdir.mkdir('config'))
-    temp_astropy_config = os.path.join(temp_config_dir, 'astropy')
+    temp_astropy_config = os.path.join(temp_config_dir, '.astropy')
 
     # Test decorator mode
     @paths.set_temp_config(temp_config_dir)
     def test_func():
-        assert paths.get_config_dir() == temp_astropy_config
+        assert paths.get_config_dir(rootname='astropy') == temp_astropy_config
```

Comment:
Theoretically, this change is not necessary for the test to pass, right? Same question for all the similar diff below.